subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1047,33 +971,40 @@ The main difference between a _dynamic version_ and a _changing module_ is that
+++++
+<sample id="changing-module-cache-control" dir="userguide/artifacts/resolutionStrategy" title="Changing module cache control">
+                <sourcefile file="build.gradle" snippet="changing-module-cache-control"/>
+            </sample>
+++++
 
-This is a common reason why many projects prefer to store their libraries in their version control system. This approach is fully supported by Gradle. The libraries can be stored in a flat directory without any XML module descriptor files. Yet Gradle offers complete transitive dependency management. You can use either client module dependencies to express the dependency relations, or artifact dependencies in case a first level dependency has no transitive dependencies. People can check out such a project from your source code control system and have everything necessary to build it.
+For more details, take a look at the API documentation for api:org.gradle.api.artifacts.ResolutionStrategy[].
 
-If you are working with a distributed version control system like Git you probably don't want to use the version control system to store libraries as people check out the whole history. But even here the flexibility of Gradle can make your life easier. For example, you can use a shared flat directory without XML descriptors and yet you can have full transitive dependency management, as described above.
+[[sec:cache_command_line_options]]
+==== Controlling dependency caching from the command line
 
-You could also have a mixed strategy. If your main concern is bad metadata in the POM file and maintaining custom XML descriptors, then _Client Modules_ offer an alternative. However, you can still use a Maven2 repo or your custom repository as a repository for _jars only_ and still enjoy _transitive_ dependency management. Or you can only provide client modules for POMs with bad metadata. For the jars and the correct POMs you still use the remote repository.
+[[sub:cache_offline]]
+===== Offline
```

Comment:
This header isn't very descriptive (and I'm not 100% sure it's necessary). If we want to keep it, perhaps "Avoiding network access with Offline Mode" would be more thorough.

Reply:
done